covid19dp_submission/steps/vcf_vertical_concat/run_vcf_vertical_concat_pipeline.py

```diff
@@ -0,0 +1,201 @@
+# Copyright 2021 EMBL - European Bioinformatics Institute
```

Comment:
At the time, we were still going to debate the use of vertical concatenation so I didn't preemptively refactor this code to a separate repository since it comes with quite an overhead. Indeed, it is an opportunity :) to the person picking up EVA-2501 and has been documented as part of the desired outcome there.